### ISSUE TITLE
What about the error tips in VSCode with ES7 async/await syntax?

### ISSUE THREAD

**Issue by wxqqh** [Microsoft/vscode#2407]
Is there VSCode full support ES7 async/await syntax?

I tried to use ES7 `async/await` syntax in VSCode. 
But i found that there was something wrong with my config, when i compiled the `*.tx` to `*.js`.

When I **compiled**, the error tips would **disappear**. But when I **modified** the file, the error tips  would **show again**.

Just liked this:

![issue](https://cloud.githubusercontent.com/assets/1575657/12605008/a8c93ad8-c4fb-11e5-9e4b-dc01bfe5ad70.gif)

-------
vscode: 0.10.6
tsc: 1.9.0-dev.20160126
node: 4.1.1

the code:
```javascript
"use strict";

async function main() {
    await ping();
}

async function ping() {
    for (var i = 0; i < 10; i++) {
        await delay(300);
        console.log("ping");
    }
    
}

function delay(ms: number) {
    return new Promise(resolve => setTimeout(resolve, ms));
}

main();
```

my config files:

tasks.json

```json
{
	"version": "0.1.0",
	"command": "tsc",
	"isShellCommand": true,
	"showOutput": "silent",
	"args": ["-p", "."],
	"problemMatcher": "$tsc"
}
```

tsconfig.json
```json
{
    "compilerOptions": {
        "target": "es2015",
        "module": "commonjs",
        "sourceMap": true,
        "outDir": "bin/",
        "moduleResolution": "classic"
    }
}
```

**Comment by dbaeumer** [comment]
VSCode ships with tsc 1.7.x. Your compiler is 1.9-dev.... This is why you see the mismatch in errors. You can make VSCode use a different tsc version as well by using the typescript.tsdk setting (see https://code.visualstudio.com/updates/vNovember)

**Comment by wxqqh** [comment]
I've tried this setting `"typescript.tsdk": "./node_modules/typescript/lib"`, but it doesn't work.

**Comment by egamma** [comment]
`typescript.tsdk` is in active use and works. 

Did you try to set an absolute path name or install typescript globally. 

In the january update Code picks up a locally installed typescript automatically.

@dbaeumer  fyi.

**Comment by dbaeumer** [comment]
@wxqqh this works for me using "typescript.tsdk": "./node_modules/typescript/lib"  and having typescript install in that workspace folder. Can you provide me with a test workspace that shows the problem.

**Comment by zhangbihua** [comment]
![tsc](https://cloud.githubusercontent.com/assets/13499245/14486477/ba69e456-018f-11e6-83a7-2462184c4e3d.png)
not work

**Comment by zhangbihua** [comment]
windown 10,
version:0.10.11

**Comment by jeremyalan** [comment]
I've been fighting this issue for a long time, and actually gave up on it for awhile, but finally decided I was going to force myself to sit down and figure it out.  Hopefully, others can benefit from this fix, and maybe the core team can offer some insight into why this is happening.

When async/await first came out, it was enabled by the following setting in `tsconfig.json`:
```
"experimentalAsyncFunctions": true
```

At some point, the feature was no longer experimental so the flag was removed, but the error in VS Code persisted (for me, anyway), until I deleted this setting completely from `tsconfig.json`.  Now, everything appears to be working as expected.

I suspect this may not be an issue for newer projects, which is why it's hard to reproduce in the latest versions of VS Code, but for those of us who dare to live on the edge, our projects still have artifacts from past implementations lurking in our configs.
